src/rules/form-control-has-label.ts

```diff
@@ -1,31 +1,38 @@
 import type { Rule } from "eslint";
 import type { AST } from "vue-eslint-parser";
 
+interface FormControlHasLabelOptions {
+  labelComponents: string[];
+}
+
 import {
   defineTemplateBodyVisitor,
   getElementAttributeValue,
   getElementType,
   hasAriaLabel,
   isAriaHidden,
-  makeDocsURL
+  makeDocsURL,
+  makeKebabCase,
 } from "../utils";
 
 function isLabelElement(
   node:
     | AST.VElement
     | AST.VDocumentFragment
     | AST.VText
-    | AST.VExpressionContainer
+    | AST.VExpressionContainer,
+  options: FormControlHasLabelOptions
 ) {
-  return node.type === "VElement" && getElementType(node) === "label";
+  const labelComponents = (options.labelComponents || []).map(makeKebabCase).concat("label");
+  return node.type === "VElement" && labelComponents.includes(getElementType(node));
```

Comment:
(resolvable) I think it's kinda of exposing implementation details having to use the `makeKebabCase` to compare stuff using `getElementType` I guess having a proper `isMatchingElement` or something could maybe better, then we wouldn't have to remember that we need to normalize stuff using `makeKebabCase` before comparing using `getElementType`, do you guys understand?

could be something like:

```js
export default function isMatchingElement(node, searchArray) {
  const elementType = getElementType(node);
  
  return searchArray.some(item => {
    return makeKebabCase(item) === elementType;
  });
}
```
But you know, just a suggestion.

Reply:
Yeah that seems cleaner, done